tools/generate_test_status.py

```diff
@@ -20,57 +20,107 @@
 Prints {{id}} to terminal.
 """
 
-from utils import error, warning, metadata
+# The name of a job is HARDCODED.
```

Comment:
We should place a comment next to all places that affect the hardcoded name, tool i.e. in the .yml files; something like "If you change this name/string/whatever,  you must adjust LIST OF PLACES THAT NEED TO BE ADJUSTED"